crates/viewer/re_ui/src/notifications.rs

```diff
@@ -0,0 +1,393 @@
+
+            self.has_unread_notifications = false;
+        }
+
+        self.panel.show(egui_ctx, &mut self.data, is_panel_visible);
+        self.toasts.show(egui_ctx, &mut self.data[..]);
+
+        if let Some(notification) = self.data.last() {
+            if !notification.ttl.is_zero() {
+                egui_ctx.request_repaint_after(notification.ttl);
+            }
+        }
+    }
+}
+
+struct NotificationPanel {
+    id: egui::Id,
+}
+
+impl NotificationPanel {
+    fn new() -> Self {
+        Self {
+            id: egui::Id::new("__notifications"),
+        }
+    }
+
+    fn show(
+        &self,
+        egui_ctx: &egui::Context,
+        notifications: &mut Vec<Notification>,
+        is_panel_visible: &mut bool,
+    ) {
+        if !*is_panel_visible {
+            return;
+        }
+
+        for notification in notifications.iter_mut() {
+            notification.ttl = Duration::ZERO;
+        }
+
+        let panel_width = 356.0;
+        let panel_max_height = 320.0;
+
+        let mut to_dismiss = None;
+
+        let notification_list = |ui: &mut egui::Ui| {
+            if notifications.is_empty() {
+                ui.label(
+                    egui::RichText::new("Nothing here!")
+                        .weak()
+                        .color(hex_color!("#636b6f")),
+                );
+
+                return;
+            }
+
+            for (i, notification) in notifications.iter().enumerate().rev() {
+                show_notification(ui, notification, DisplayMode::Panel, || {
+                    to_dismiss = Some(i);
+                });
+            }
+        };
+
+        let mut clear = false;
+
+        egui::Area::new(self.id)
+            .anchor(egui::Align2::RIGHT_TOP, egui::vec2(-8.0, 32.0))
+            .order(egui::Order::Foreground)
+            .interactable(true)
+            .movable(false)
+            .show(egui_ctx, |ui| {
+                egui::Frame::window(ui.style())
+                    .fill(hex_color!("#141819"))
+                    .rounding(8.0)
+                    .inner_margin(8.0)
+                    .show(ui, |ui| {
+                        ui.set_width(panel_width);
+                        ui.horizontal_top(|ui| {
+                            ui.label("Notifications");
+                            ui.with_layout(egui::Layout::top_down(egui::Align::Max), |ui| {
+                                if ui.small_icon_button(&icons::CLOSE).clicked() {
+                                    *is_panel_visible = false;
+                                }
+                            });
+                        });
+                        egui::ScrollArea::vertical()
+                            .min_scrolled_height(panel_max_height)
+                            .show(ui, notification_list);
+
+                        if !notifications.is_empty() {
+                            ui.horizontal_top(|ui| {
+                                if ui.button("Clear all").clicked() {
```

Comment:
```suggestion
                                if ui.button("Dismiss all").clicked() {
```

to match the wording used per-notification